README.md

```diff
@@ -77,7 +77,7 @@ TensorRT and OpenVino support:
 ```
 $ mkdir build
 $ cd build
-$ cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install -DTRITON_BUILD_ONNXRUNTIME_VERSION=1.8.0 -DTRITON_BUILD_CONTAINER_VERSION=21.05 -DTRITON_ENABLE_ONNXRUNTIME_TENSORRT=ON -DTRITON_ENABLE_ONNXRUNTIME_OPENVINO=ON -DTRITON_BUILD_ONNXRUNTIME_OPENVINO_VERSION=2021.2.200 ..
+$ cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install -DTRITON_BUILD_ONNXRUNTIME_VERSION=1.8.1 -DTRITON_BUILD_CONTAINER_VERSION=21.07 -DTRITON_ENABLE_ONNXRUNTIME_TENSORRT=ON -DTRITON_ENABLE_ONNXRUNTIME_OPENVINO=ON -DTRITON_BUILD_ONNXRUNTIME_OPENVINO_VERSION=2021.2.200 ..
```

Comment:
Which ORT release that OpenVINO 2021.4 will be used? We wanted to bump the version in OpenVINO backend but we paused the PR to align with ORT.

Reply:
Support for Openvino 2021.4 is already part of ORT master and will be part of ORT 1.9 release. Since we are picking from master I can update Openvino to 2021.4... let me know

Reply:
@GuanLuo  @deadeyegoodwin  This will be great. There is some interest on openVINO backend and moving to 2021.4 will help us improve performance and run proper testing to move the backend outside beta.

Reply:
@askhade Yea, it would be great if we update OpenVINO to 2021.4 in this PR as well

Reply:
Actually, can we do openvino separately... OV needs to be updated consistently across both ORT backend and OV backend. We must get the TRT8 support in 21.08 but getting OV to 2021.4 is not a blocker for 21.08 so I'd like to keep them separate.

Reply:
Sounds good. I will send out the PR to update OV after 21.08 code freeze. Then we can checkin both the PRs in a coordinated fashion.